lib/rspec/matchers/dsl.rb

```diff
@@ -336,7 +340,7 @@ class Matcher
         # Allows expectation expressions to be used in the match block.
         include RSpec::Matchers
 
-        # Converts matcher name and expected args to an English expresion.
+        # Facilitates better descriptions and failure messages.
         include RSpec::Matchers::Pretty
```

Comment:
> However, I think we can make Pretty#underscore private, and I have done so.

On second thought, though `Pretty#underscore` was designated `@api private`, I don't know exactly what that means in this project, so making `#underscore` private may or may not constitute a breaking change.  Please advise.